package/MDAnalysis/coordinates/PDB.py

```diff
@@ -691,18 +680,23 @@ class PrimitivePDBWriter(base.Writer):
     # x)8.3f%(y)8.3f%(z)8.3f%(occupancy)6.2f%(tempFactor)6.2f          %(element)2s%(charge)2d\n",
     # PDB format as used by NAMD/CHARMM: 4-letter resnames and segID, altLoc ignored
     fmt = {
-        'ATOM': "ATOM  %(serial)5d %(name)-4s%(altLoc)-1s%(resName)-4s%(chainID)1s%(resSeq)4d%(iCode)1s   %(x)8.3f%("
-                "y)8.3f%(z)8.3f%(occupancy)6.2f%(tempFactor)6.2f      %(segID)-4s%(element)2s%(charge)2d\n",
-        'REMARK': "REMARK     %s\n",
-        'COMPND': "COMPND    %s\n",
-        'HEADER': "HEADER    %s\n",
-        'TITLE': "TITLE     %s\n",
-        'MODEL': "MODEL     %5d\n",
-        'NUMMDL': "NUMMDL    %5d\n",
+        'ATOM': (
+            "ATOM  {serial:5d} {name:<4s}{altLoc:<1s}{resName:<4s}"
+            "{chainID:1s}{resSeq:4d}{iCode:1s}"
+            "   {pos[0]:8.3f}{pos[1]:8.3f}{pos[2]:8.3f}{occupancy:6.2f}"
+            "{tempFactor:6.2f}      {segID:<4s}{element:2s}{charge:.2f}\n"),
```

Comment:
http://www.wwpdb.org/documentation/file-format-content/format33/sect9.html#ATOM

This is the reference I use for looking up the PDB format.  I can't find the extra `segid` field you introduced here between `tempfactor` and `element`. Do I use an outdated reference or is this some unofficial add on?